src/types/pdfjs.ts

```diff
@@ -66,6 +66,13 @@ export type PDFDocument = {
    */
   fingerprints?: [string, string | null];
   getMetadata(): Promise<PDFDocumentMetadata>;
+
+  /**
+   * @return A promise that is resolved when the document's data is loaded.
+   *         It is resolved with an {Object} that contains the `length` property
+   *         that indicates size of the PDF data in bytes.
```

Comment:
I basically copied the JSDoc from PDF.js repo https://github.com/mozilla/pdf.js/blob/a1b45d6e69d37b76e675a7f247943b70f186baca/src/display/api.js#L1024-L1028